src/index.html

```diff
@@ -18,5 +22,85 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="/apple"
+              class="nav_link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="/samsung"
+              class="nav_link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="/smartphones"
+              class="nav_link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="/laptops"
+              class="nav_link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="/gadgets"
+              class="nav_link hover"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="/tablets"
+              class="nav_link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="/photo"
+              class="nav_link"
+            >
+              PHOTO
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be internal links.